bld/test/ut/libpal/CMakeLists.txt

```diff
@@ -17,21 +17,21 @@ set(libpal_ut_include
 # Misc pal components to test
 #
 set(_MISC_PAL_UNITS_UNDER_TEST
-    new__pal_sk_posix
+    pal_sk_posix
```

Comment:
Yep :-), same as above, this is a partial one which unfortunately has some extra cmakelists.txt changes that should not have come in.